extensions/sql-migration/src/wizard/assessmentDetailsPage/assessmentDetailsHeader.ts

```diff
@@ -207,6 +208,9 @@ export class AssessmentDetailsHeader {
 					if (hasRecommendations(this.migrationStateModel)) {
 						const generateProvisioningScriptDialog = new GenerateProvisioningScriptDialog(this.migrationStateModel, this.migrationStateModel._targetType);
 						await generateProvisioningScriptDialog.openDialog();
+						if (this.migrationStateModel._targetType === MigrationTargetType.SQLDB) {
+							await vscode.window.showInformationMessage(constants.DISPLAY_ARM_TEMPLATE_LIMIT);
```

Comment:
We should show this msg only if we have more than 1 template, not in other case.

Reply:
done